### PR TITLE
added better mocking of new endpoints

### PR DESCRIPTION
**Pull request by Manoj-nathwani**
- Added better mocking to `main.py` endpoints to allow the UI to be more user friendly
- Made small changes to a few data structures and names of things

### CODE REVIEW

File: navigator_api/main.py [hunk 1]
```diff
@@ -70,58 +71,40 @@ def workflow_state(dataset_id):
...
-        "content": {
-            "title": "Populate geo template",
-            "display_html": "<p>Lorem Ipsum has been the industry's standard dummy text ever since the <strong>1500s</strong></p>",
-            "skippable": True,
-            "url": "http://fjelltopp.org"
+                "display_html": mockup.html(),
+                "actionable_links": mockup.actionable_links()
+            },
         }
     })
```
Comment (Manoj-nathwani) [inline navigator_api/main.py:109]: Please confirm if I am safe removing this. I wasn't sure why we had it as `/workflows/<dataset_id>/state` already gives us the same data

File: navigator_api/main.py [hunk 2]
```diff
@@ -70,58 +71,40 @@ def workflow_state(dataset_id):
         "id": "xxx-yyy-zzz",
         "milestones": [
             {
-                "id": "xxx",
-                "title": "Naomi Data Prep",
+                "id": mockup.id(),
+                "title": mockup.milestone(),
```
Comment (tomeksabala) [inline navigator_api/main.py:75]: This is overthinking IMO. We should keep static/hardcoded content here. I don't see any value in introducing the `utils.mockup` module. Those stubs will go away in a couple of days after nav_engine gets a working API.

File: navigator_api/main.py [hunk 3]
```diff
@@ -70,58 +71,40 @@ def workflow_state(dataset_id):
...
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
+            mockup.id(),
         ],
-        "currentTask": {
-            "id": "asdfasdfa",
-            "skipped": False,
+        "current_task": {
```
Comment (tomeksabala) [inline navigator_api/main.py:100]: I have decided to go with camelCase key names in json structures according to google recommentations https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format

File: navigator_api/main.py [hunk 4]
```diff
@@ -70,58 +71,40 @@ def workflow_state(dataset_id):
...
+            "id": mockup.id(),
+            "milestone_id": mockup.id(),
```
Comment (tomeksabala) [inline navigator_api/main.py:102]: adding milestone id in task content probably makes sense. We should talk about it with the team as it requires changes in engine docs.

File: navigator_api/main.py (hunk 1 above)
Comment (tomeksabala) [reply navigator_api/main.py:109]: We were discussing this on a call what's the purpose of this endpoint and why it exists. It's not safe to remove this.
It should be used to get data about previous tasks.

**Review by tomeksabala — CHANGES_REQUESTED** [review]
I think that this PR only complicates things with very little added value.
We should discuss potential changes in the content of the responses with the team.
Let's discuss needed changes to the api design and I'll submit the changes in a separate PR.

**Pull request closed without merging**